examples/serial-blocking.toit

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) 2025 Toitware ApS. All rights reserved.
+// Use of this source code is governed by a MIT-style license that can be found
+// in the LICENSE file.
+
+import gpio
+import i2c
+import uart
+import io
+
+import ublox-gnss
+
+TX-PIN := gpio.Pin 17
+RX-PIN := gpio.Pin 18
+BAUD   := 9600
+UART   := 0
+
+main:
+  port := uart.Port --tx=TX-PIN --rx=RX-PIN --baud-rate=BAUD
+
+  driver := ublox-gnss.Driver port.in port.out
+
+  print "getting location"
+
+  driver.location --blocking
+  print "Took: $(driver.time-to-first-fix)"
+
+  while true:
+    location := driver.location --blocking
```

Comment:
Add a sleep. The gps modules probably rate limits itself, but it still feels cleaner.